### ISSUE TITLE
Pin length inconsistency between view and password count

### ISSUE THREAD

**Issue by Mostafa3la2** [xornorik/SVPinView#25]
I had this issue in some projects where setting pin length to another length rather than the default value would update the view but the password count is still 5 leading to either crashes when the pin length is greater than the default value or not triggering didFinishCallback if less than 5.
I fixed this issue by moving the password array initialization to the collection view delegate method numberOfItemsInSection rather than in loadView that gets called in init method.

SVPin version : 1.0.6
Deployment Target : iOS 11
 it doesn't appear in iOS 12.1 projects though.

**Comment by ssowri1** [comment]
Yes. Its too annoying to me. I am getting same error. My configuration in below,

 ```
       mobileOtp.pinLength = 4
        mobileOtp.didFinishCallback = didFinishEnteringPin(pin:)
        mobileOtp.didChangeCallback = { pin in
            print("The mobileOtp pin is \(pin)")
        }
    
func didFinishEnteringPin(pin: String) {
        print("mobileOtp entered is \(pin)")
        self.showAlertWithTitle(title: "", message: pin)
    }

```

My console output is,
~~~~["5", "3", "8", "3", ""]

Therefore my **mobileOtp.didFinishCallback** could not called when I reach 4 digit password.

My deployment target is **11.0**
iOS : 12.1.4

Please share the solution to get rid of this issue. Thanks in advance.

**Comment by xornorik** [comment]
@Mostafa3la2 @ssowri1 Apologies for the delay, have fixed the issue and made a release (1.0.7). Do let me know in case of any further issues. 
Thanks.

**Comment by ssowri1** [comment]
@xornorik: Thank you for your support.
